moonscript.html.markdown

```diff
@@ -0,0 +1,570 @@
+---
+language: moonscript
+contributors:
+  - ["RyanSquared", "https://ryansquared.github.io/"]
+  - ["Job van der Zwan", "https://github.com/JobLeonard"]
+filename: moonscript.moon
+---
+
+MoonScript is a dynamic scripting language that compiles into Lua. It gives
+you the power of one of the fastest scripting languages combined with a
+rich set of features.
+
+See [the MoonScript website](https://moonscript.org/) to see official guides on installation for all platforms.
+
+```moonscript
```

Comment:
Please correct this to

```
```moon
```

In its present state, this will cause the lexer to fail, therefore the page build will fail and it'll return a 404 on the site.